File: work_android_network/ws_android_network_04_3_SSAFYGo_4/src/main/java/com/ssafy/ssafygo/model/service/StoreMenuService.java
package com.ssafy.ssafygo.model.service;

import java.util.List;

import com.ssafy.ssafygo.model.dto.StoreMenu;

public interface StoreMenuService {
	// 해당 가맹점 메뉴 조회
    public List<StoreMenu> findAllStoreMenuByStoreId(String storeId);

    // 해당 메뉴 조회
    public StoreMenu findStoreMenuByMenuId(String menuId);
}

File: work_android_network/ws_android_network_03_3_leegideok/app/src/main/java/com/ssafy/ssafygo/service/StoreMenuService.kt
package com.ssafy.ssafygo.service

import com.ssafy.ssafygo.dto.StoreMenuDTO
import com.ssafy.ssafygo.dto.StoreReviewDTO
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface StoreMenuService {
    // 매뉴 정보 전부 가져오기
    @GET("store/{storeId}/menus")
    fun getStoreMenus(@Path("storeId") storeId: Int): Call<List<StoreMenuDTO>>

    // 해당 매뉴 정보 가져오기
    @GET("store/menu/{menuId}")
    fun getStoreMenu(@Path("menuId") reviewId: Int): Call<StoreMenuDTO>

}